PhysicsTools/NanoAOD/plugins/GlobalVariablesTableProducer.cc

```diff
@@ -40,6 +43,26 @@ class GlobalVariablesTableProducer : public edm::stream::EDProducer<> {
 
   ~GlobalVariablesTableProducer() override {}
 
+  static void fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+    edm::ParameterSetDescription desc;
+    desc.addOptional<std::string>("name")->setComment("name of the branch in the flat table output");
+    desc.addOptional<bool>("extension", false)->setComment("whether or not to extend an existing same table");
+    edm::ParameterSetDescription variable;
+    variable.ifValue(edm::ParameterDescription<std::string>(
+                         "type", "int", true, edm::Comment("the c++ type of the branch in the flat table")),
+                     edm::allowedValues<std::string>(
+                         "int", "float", "double", "bool", "candidatescalarsum", "candidatesize", "candidatesummass"));
+    variable.add<edm::InputTag>("src")->setComment("input collection for the branch");
+    variable.add<std::string>("doc")->setComment("few words description of the branch content");
+    variable.addOptional<int>("precision")->setComment("precision to store the information [NOT USED IN THE CODE]");
```

Comment:
N.B. this is added to avoid breaking existing configurations specifying `precision` even though it is not used. a subsequent PR will add the usage of precision for `GlobalVariablesTableProducer`